ludwig/progress_bar.py

```diff
@@ -0,0 +1,73 @@
+import uuid
+
+import tqdm
+
+try:
+    import ray.train as rt  # noqa: E402
```

Comment:
Is the `noqa: E402` necessary?

Reply:
I though E402 also applies if it is wrapped in a try except

Reply:
It seems like it may not be, i.e. https://github.com/ludwig-ai/ludwig/blob/master/ludwig/utils/data_utils.py#L45-L52

One way to test whether the annotation is needed (I think) is to try removing the annotation and see if flake8 (should be enabled automatically in VSCode for the Ludwig repo) complains.

Reply:
You are right my friend